templates/bundles/PrestaShopBundle/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -1221,3 +1221,34 @@
   {{- block('form_widget_simple') -}}
   <img src="{{ form.vars.value }}" alt="{{ ('Image preview for ' ~ form.vars.name)|trim }}" />
 {% endblock %}
+
+{% block delta_quantity_widget %}
+  {%- set attr = attr|merge({class: (attr.class|default('') ~ ' delta-quantity')|trim, 'data-initial-quantity': form.quantity.vars.value}) -%}
+  <div {{ block('widget_container_attributes') }}>
+    {{ form_widget(form.quantity) }}
+    {{ form_row(form.delta) }}
+  </div>
+{% endblock %}
+
+{% block delta_quantity_quantity_widget %}
+  {%- set attr = attr|merge({class: (attr.class|default('') ~ ' delta-quantity-quantity')|trim}) -%}
+  <div {{ block('widget_container_attributes') }}>
+    {{- block('form_widget_simple') -}}
+    <span class="initial-quantity">
+      {{ form.vars.value|e('html') }}
```

Comment:
Isn't it the default value ? `|e('html')`

I think
```
{{ form.vars.value|e('html') }}
```
is equivalent to
```
{{ form.vars.value }}
```
if Twig [autoescape](https://twig.symfony.com/doc/3.x/api.html) is turned on

Reply:
Alright I'll remove it then